docs/installing-northstar/troubleshooting.md

```diff
@@ -151,11 +131,10 @@ The real error is most likely slightly higher. Please post that in issues or the
 
 ## I can't play the Campaign <a href="#campaign" id="campaign"></a>
 
-Campaign is not supported in Northstar yet, but you could do one of these "_**fixes**_"
+If having trouble playing the campaign, try the following fixes
 
-1. Doing `ns_auth_allow_insecure 1` in the console
-   * [If you can't open the console](troubleshooting.md#console)
-2. Adding `-vanilla` to your launch options
+1. Update your Northstar install to the [Newest Release](https://github.com/R2Northstar/Northstar/releases)
+2. Add `-vanilla` to your launch options and play vanilla Titanfall
```

Comment:
We should just drop the `-vanilla` mention cause that literally just launches vanilla Titanfall2 without any Northstar modifications. At this point, one can just launch vanilla Titanfall2 directly :P

Reply:
I'm not too sure how github works and to concretely mention changes I've made but I'll do it through this for now: Vanilla mention has been dropped, as well as the unnecessary 1. following the information to update